nimbus.fml.yaml

```diff
@@ -181,6 +185,15 @@ features:
             EXPIRES_QUICKLY:
               priority: 100
               max-display-count: 1
+      - channel: developer
```

Comment:
is the new `channel: developer` notation required to add the value, or can this be included in the above `channel: developer`?